server/implementation/src/main/java/io/smallrye/graphql/bootstrap/Bootstrap.java

```diff
@@ -237,7 +237,7 @@ private void createGraphQLInterfaceType(InterfaceType interfaceType) {
         GraphQLInterfaceType graphQLInterfaceType = interfaceTypeBuilder.build();
         // To resolve the concrete class
         this.codeRegistryBuilder.typeResolver(graphQLInterfaceType, new InterfaceResolver(interfaceType));
-        this.interfaceMap.put(interfaceType.getClassName(), graphQLInterfaceType);
+        this.interfaceMap.put(interfaceType.getName(), graphQLInterfaceType);
```

Comment:
Why did this change from classname to name ? (Just out of interest, seems like everything is still working...)

Reply:
Even interfaces can be generic, so graphql name is unique, but class name may not be. So better to use name here